### ISSUE TITLE
Hello NewMoon,  i want to join the darkside team

### ISSUE THREAD

**Issue by HyperDogs** [N3WM00N/play_class#2]
I need opportunity plz.

**Comment by tetaez01** [comment]
i need somebody to love.

**Comment by N3WM00N** [comment]
r u fu*king kidding me?

**Comment by HyperDogs** [comment]
Noooooooooooooooooo!

**Comment by HyperDogs** [comment]
"No Ordinary Love"

Here I stand,
Nothing more than a broken man starring death in the face.
All my mistakes have got a hold on me again.

Here we go,
Another day, another fork in the road.
I never know who to trust,
Will someone just tell me which way to go?

I need a sign.
I need a miracle!
I need to know that this is possible.
I've just failed so many times,
Is there any way that I could make it right?
God, if you're there I need to hear you now!

Come back to me love, I forgive you.
Oh how I've missed you! (I've missed you)
Just know every time you run I'll be right here waiting!

Yeah!

This is no ordinary love.
Listen up, Listen up!
This is no ordinary love.
Listen up this is unconditional!

I'm so afraid to admit that you're real
Cause I'm afraid of what I might feel
When your words start to come to life
And it brings my demons into the light but I know you're here.
I shiver as you whisper in my ear, "Child have no fear".

Come back to me love, I forgive you.
Oh how I've missed you! (I've missed you)
Just know every time you run I'll be right here waiting!
Come back to me love.
I hope you know I'm always with you. (I'm always with you)
Oh, I forgive you!

We lose faith in things unseen,
Unsure of what to believe,
Just dying to be loved when love's right in front of us.
We give up on our hopes & dreams, & let doubt take everything.
The door will open when we realize we don't hold the key.

I tried to save myself but I failed!
But I failed!

Come back to me love, I forgive you.
Oh how I've missed you! (I've missed you so)
Every time you run I'll be right here waiting!
Come back to me love.
I hope you know I'm always with you. (I'm always with you)
I forgive you!
